src/main/java/gyro/google/compute/InstanceAttachedDiskResource.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+import com.google.api.client.googleapis.json.GoogleJsonError;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.services.compute.Compute;
+import com.google.api.services.compute.model.AttachedDisk;
+import com.google.api.services.compute.model.Instance;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.Wait;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Required;
+import gyro.google.Copyable;
+
+/**
+ * Attaches a disk to an instance.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *      google::instance-attached-disk gyro-disk-1
+ *          instance: $(google::instance gyro-dev-1)
+ *          attached-disk
+ *              auto-delete: false
+ *              source: $(google::compute-disk instance-disk-example)
+ *          end
+ *      end
+ */
+@Type("instance-attached-disk")
```

Comment:
Prefix this with `compute-` and update the examples with the new type:
```suggestion
@Type("compute-instance-attached-disk")
```

Reply:
Done.